blueprints/ember-cli-blanket/index.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
   normalizeEntityName: function() {},
 
   afterInstall: function() {
-    return this.addBowerPackageToProject('blanket', '~1.1.5', {saveDev: true})
+    return this.addBowerPackageToProject('blanket', '#5e94fc30f2e694bb5c3718ddcbf60d467f4b4d26', {saveDev: true})
```

Comment:
I've hashed through this a few times in my mind and with code.  In short, if you've installed this version of ember cli-blanket you need to have that version of blanket.js installed.  Full Stop.   There is no backward compatible integration with older versions of loader.js  we should consider this when changing the version number.

I have the code for the blueprint done but in practice ember cli calls init on addons as it collects them - that's where we are doing the version check so we emit an error and bomb out there well before you ever get to blueprint generation.